tests/sentry/tsdb/test_redis.py

```diff
@@ -25,8 +25,15 @@ def setUp(self):
             ),
             vnodes=64,
             enable_frequency_sketches=True,
+            hosts={
+                i - 6: {'db': i} for i in xrange(6, 9)
+            },
         )
 
+    def tearDown(self):
+        with self.db.cluster.fanout('all') as client:
```

Comment:
No specific reason to prefer one over the other, I just went with the one that I remembered exists. I changed it to `.all` because it seems more reliable to use a method here.